packages/terra-modal/src/modal.scss

```diff
@@ -0,0 +1,79 @@
+.terra-Modal-backdrop {
+  background-color: #1c1f21;
+  border: 0;
+  height: 100%;
+  left: 0;
+  opacity: 0.8;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1000;
+}
+
+.terra-Modal-dialog {
+  background-color: #fff;
+  border-radius: 0.25em;
+  left: 0;
+  margin: 0 auto;
+  outline: 0;
+  position: fixed;
+  right: 0;
+  top: 5%;
+  width: 85%;
+  z-index: 1001;
```

Comment:
will update this when this pull request is merged: cerner/terra-legacy-theme#12.